modules/ROOT/assets/attachments/sync-gateway-admin.yaml

```diff
@@ -386,13 +386,21 @@ parameters:
         HTTP:
           type: boolean
           description: All requests made to the Sync Gateway REST APIs (Sync and Admin). Note that the log keyword HTTP is always enabled, which means that HTTP requests and error responses are always logged (in a non-verbose manner). HTTP+ provides more verbose HTTP logging.
-  loglevel:
+  level:
     in: query
     name: level
     description: |
       Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
       panics logging and level=3 will log panics only.
     type: integer
+    deprecated: true
+  logLevel:
+    in: query
+    name: logLevel
+    description: |
+      Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
```

Comment:
The parameter was changed to accept the same textual values as the SG config. Valid options are `none`, `error`, `warn`, `info`, `debug`, `trace`:

https://github.com/couchbase/docs-sync-gateway/blob/8b15b8fbd002d17636d9099eed06780be847c6f0/modules/ROOT/pages/logging.adoc#log-levels